src/components/GroceryItem/GroceryItem.js

```diff
@@ -47,16 +48,49 @@ const GroceryItem = ({ item }) => {
       });
   };
 
-  const resetDate = () => {
+  // Should we delete the option to uncheck? Otherwise, we can't really reset the previousPurchaseDate in Firestore.
+  const handleUncheckBox = () => {
     setChecked(false);
   };
 
+  const getAriaLabel = () => {
+    if (
+      currentDate - item.purchaseDate >=
+        2 * (item.nextPurchaseDate - item.purchaseDate) ||
+      item.numberOfPurchases === 0
+    ) {
+      return 'Inactive';
+    } else if (item.howSoon < 7) {
+      return 'Need to buy soon';
+    } else if (item.howSoon >= 7 && item.howSoon <= 30) {
+      return 'Need to buy kind of soon';
+    } else if (item.howSoon > 30) {
+      return 'Do not need to buy soon';
+    }
+  };
+
+  const getBackground = () => {
+    if (
+      currentDate - item.purchaseDate >=
+        2 * (item.nextPurchaseDate - item.purchaseDate) ||
+      item.numberOfPurchases === 0
+    ) {
+      return 'background-grey';
+    } else if (item.howSoon < 7) {
+      return 'background-red';
+    } else if (item.howSoon >= 7 && item.howSoon <= 30) {
+      return 'background-yellow';
+    } else if (item.howSoon > 30) {
+      return 'background-green';
+    }
+  };
+
   return (
     <>
-      <p>
+      <p aria-label={getAriaLabel()} className={getBackground()}>
```

Comment:
Thanks for adding ARIA here! Just couple of observations: `aria-label` should be included in the `input` element instead of the `p` tag. `p` might have a problem with compatibility across assistive technologies (e.g. screen readers).

This is the current implementation (`aria-label` applied to `p`):
<img width="1113" alt="Screen Shot 2021-05-13 at 10 13 13 PM" src="https://user-images.githubusercontent.com/843075/118211313-fca8f980-b439-11eb-9c41-97e08adbfec9.png">

This is how it would look if you apply aria to the `input`:
<img width="993" alt="Screen Shot 2021-05-13 at 10 17 28 PM" src="https://user-images.githubusercontent.com/843075/118211564-4e518400-b43a-11eb-8b83-4776aa5548d3.png">

Another thing that noticed is that the screen reader would read this as `Need to buy soon` but the user might be missing the product name. So maybe something like `corn: Need to buy soon` or `Need to buy corn soon` would be more accessible for the person using it :)

**NOTE:** I used Mac VoiceOver to test this feature, so maybe this would be different in another screen reader (e.g. JAWS). 

You can find more info here: https://www.tpgi.com/short-note-on-aria-label-aria-labelledby-and-aria-describedby/